build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -134,6 +137,16 @@ class SeleniumWebDriverController {
     return webElements.map(webElement => new ElementHandle(webElement, this));
   }
 
+  /**
+   * @return {!Promise<!ElementHandle<!WebElement>>}
+   * @override
+   */
+  async getActiveElement() {
```

Comment:
Seems like we would want a version of this that returns the active element for the current document or the shadow root instead of always for the document.

Reply:
That's a good idea. I will implement the shadow root functionality for `getActiveElement` after I merge a PR I have in progress that enables tests in the shadow-viewer.